ui/src/components/WMWorkersTable/index.jsx

```diff
@@ -0,0 +1,257 @@
+            />
+            <LinkIcon size={iconSize} />
+          </TableCellListItem>
+        </TableCell>
+
+        <CopyToClipboard title={`${workerAge} (Copy)`} text={workerAge}>
+          <TableCell>
+            <TableCellListItem button>
+              <ListItemText
+                disableTypography
+                primary={
+                  <Typography>
+                    <DateDistance from={workerAge} />
+                  </Typography>
+                }
+              />
+              <ContentCopyIcon size={iconSize} />
+            </TableCellListItem>
+          </TableCell>
+        </CopyToClipboard>
+
+        <TableCell>
+          {latestTask ? (
+            <TableCellListItem
+              button
+              component={Link}
+              to={`/tasks/${latestTask.run.taskId}/runs/${
+                latestTask.run.runId
+              }`}>
+              <ListItemText
+                disableTypography
+                primary={<Typography>{latestTask.run.taskId}</Typography>}
+              />
+              <LinkIcon size={iconSize} />
+            </TableCellListItem>
+          ) : (
+            <Typography>n/a</Typography>
+          )}
+        </TableCell>
+
+        {latestTask ? (
+          <CopyToClipboard
+            title={`${latestTask.run.started} (Copy)`}
+            text={latestTask.run.started}>
+            <TableCell>
+              <TableCellListItem button>
+                <ListItemText
+                  disableTypography
+                  primary={
+                    <Typography>
+                      <DateDistance from={latestTask.run.started} />
+                    </Typography>
+                  }
+                />
+                <ContentCopyIcon size={iconSize} />
+              </TableCellListItem>
+            </TableCell>
+          </CopyToClipboard>
+        ) : (
+          <TableCell>
+            <Typography>n/a</Typography>
+          </TableCell>
+        )}
+
+        {latestTask ? (
+          <CopyToClipboard
+            title={`${latestTask.run.resolved} (Copy)`}
+            text={latestTask.run.resolved}>
+            <TableCell>
+              <TableCellListItem button>
+                <ListItemText
+                  disableTypography
+                  primary={
+                    <Typography>
+                      <DateDistance from={latestTask.run.resolved} />
+                    </Typography>
+                  }
+                />
+                <ContentCopyIcon size={iconSize} />
+              </TableCellListItem>
+            </TableCell>
+          </CopyToClipboard>
+        ) : (
+          <TableCell>
+            <Typography>n/a</Typography>
+          </TableCell>
+        )}
+
+        <TableCell>
+          <TableCellListItem
+            button
+            component={Link}
+            to={`${path}/worker-types/${workerType}/workers/${workerGroup}/${workerId}/recent-errors`}>
```

Comment:
Clicking turns the url into `http://localhost:5080/worker-manager/worker-types/:workerType/worker-types/banana/workers/%F0%9F%A6%80/rust-awesomness/recent-errors`. `worker-types` seems to be duplicated and there is `:workerType`.

Reply:
Oh, that link is supposed to direct you to the list of the recent tasks for the worker. I'll remove the link altogether for now